test/data/mockgen.js

```diff
@@ -0,0 +1,45 @@
+/*****
```

Comment:
Why are you are re adding the mocked tests that you removed on the last PR?

Reply:
It is required in both: test/data/settlements/{settlementId}/participants/{participantId}.js and test/data/settlements/{settlementId}/participants/{participantId}/accounts/{accountId}.js